### PR TITLE
docs: added readme, updated run command for sample

### PR DESCRIPTION
**Pull request by salsal97**
Puts a top level readme for the samples directory in SGX-LKL & provides a short explanation of the samples

### CODE REVIEW

**Review by paulcallen — APPROVED** [review]

File: samples/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+### Sample programs to demonstrate SGX-LKL functionality
+
+1. Basic
+   
+   a. [Attack](basic/attack) <br>
```
Comment (letmaik) [inline samples/README.md]: Can we really not do without HTML in this Markdown file?

Comment (salsal97) [reply samples/README.md]: Changed the file to remove HTML

File: samples/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        Test the two scenarios and see how i) SGX-LKL protects data as compared to regular containers and ii) SGX-LKL hardware provides protection to the data in it
+
+   b. [Helloworld](basic/helloworld) \
+        Cross-compile a C application for running with SGX-LKL
+        Simple hello world C program run in sgx-lkl
+    
+2. Containers
+   
+    a. [Alpine](containers/alpine) \
+        Miniroot is a quick way of building a small Alpine disk image, appropriate for use with sgx-musl hosting redis.
```
Comment (vtikoo) [inline samples/README.md]: Do you mean sgx-lkl or sgx-musl?

Comment (salsal97) [reply samples/README.md]: Fixed the mixup in language, thanks for pointing that out!

**Review by vtikoo — APPROVED** [review]
LGTM

**Pull request merged**